exercises/simple-linked-list/Example.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+public class SimpleLinkedList<T>
+{
+    public SimpleLinkedList(T value) :
+        this(value, null)
+    {
+    }
+
+    public SimpleLinkedList(T value, SimpleLinkedList<T> next)
+    {
+        Value = value;
+        Next = next;
+    }
+
+    public T Value { get; }
+
+    public SimpleLinkedList<T> Next { get; }
+
+    public static SimpleLinkedList<T> FromEnumerable(IEnumerable<T> values)
```

Comment:
Your call. The more I think about implementing `IList`, the more I realize that it might not make sense for a linked list (since it has methods that involve indices that linked lists don't really support well). However, `IList` implements `ICollection` which might make more sense.